pep-0001.txt

```diff
@@ -737,12 +739,10 @@ Once the PEP is ready for the repository, a PEP editor will:
   ("Standards Track", "Informational", or "Process"), and marked its
   status as "Draft".
 
-* Add the PEP to a local fork of the PEP repository.  For workflow
-  instructions, follow `The Python Developers Guide <http://docs.python.org/devguide>`_
-
-  The git repo for the peps is::
+* Add the PEP to a local fork of the `PEP repository`_.  For workflow
+  instructions, follow `The Python Developers Guide <https://devguide.python.org/>`_
 
-   https://github.com/python/peps
+  The Git repo for the peps is https://github.com/python/peps
```

Comment:
If we're changing this line, might as well update the capitalisation to be consistent.
```suggestion
  The Git repo for the PEPs is https://github.com/python/peps
```

Reply:
```suggestion
```

I suggest simply eliminating this line, since the PEP repo is already references and directly linked a couple lines above at this same URL.